platform/broadcom/docker-syncd-brcm/bcmsh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+/usr/bin/socat - UNIX-CONNECT:/var/run/sswsyncd/sswsyncd.socket
```

Comment:
I'd like you to add in the base image.

Reply:
The domain socket file is inside docker-syncd-brcm.
There is no domain socket at all on other platform.
So this wrapper is only useful here.

---
In reply to: [184841807](https://github.com/Azure/sonic-buildimage/pull/1657#discussion_r184841807) [](ancestors = 184841807)

Reply:
under /var/run/docker-syncd

Reply:
you can have another wrapper script in base image to do docker exec -it ....

Reply:
I realized that you are talking about debian base image. I prefer keep it inside brcm-syncd since it is a local specific troubleshooting tool, and not expected to be exposed to host users. The same idea is applied to platform provided dump/dbg/... tools.


---
In reply to: [184862817](https://github.com/Azure/sonic-buildimage/pull/1657#discussion_r184862817) [](ancestors = 184862817)

Reply:
If we think this socket is only available in brcm platform, we should keep it in docker-syncd-brcm only.
and if we need make the command available at host side, we should add bcmsh to
platform/broadcom/docker-syncd-brcm/base_image_files/
with wrap command in it:
docker exec -i syncd xx